scripts/test_e2e.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# test_e2e.sh
+# Execute end-to-end (e2e) tests to verify that everything is working right
+# from the end user perpsective
+set -xeuo pipefail
```

Comment:
@carolynvs 
I do not know what version of bash travisCI uses,
but Bash version 4.3 and older doesn't handle `set -u` very well in some circumstances(arrays)
But I don't think you are using arrays so, this may be fine

https://github.com/anordal/shellharden/blob/master/how_to_do_things_safely_in_bash.md#how-to-begin-a-bash-script